app/Game/game.cpp

```diff
@@ -142,48 +193,13 @@ int Game::run()
 bool Game::initSnake(Point begin, int dir, int length)
 {
     bool flag = snake.init(begin, dir, length);
-    if (!flag){
-        labyrinth.addSnake(&snake);
-    }
-    return flag;
-}
-
-/**
- * @brief   Ball initialization with labyrinth updating
- * @param   ball - a pointer to a Ball, which is being initialized
- * @return       - mark of succesful initialization
- */
-bool Game::initBall(Ball* ball)
-{
-    ball->init(gameFieldSize);
-    bool flag = ball->generateBall(&labyrinth, change, changeSize);
-    labyrinth.addPoint(ball->getCoords());
-    return flag;
-}
-
-/**
- * @brief   Initialization of the change array
- * @param   lChange - The pointer to an array
- * @param   size   - The size of array(number of elements it can contain)
- */
-bool Game::wipeChange(Point** lChange, int size)
-{
-    Point emptyPoint;
-    emptyPoint.x = -1;
-    emptyPoint.y = -1;
-    emptyPoint.style.bg = 0;
-    emptyPoint.style.fg = 0;
-    emptyPoint.style.letter = 0;
-    for(int i = 0; i < size; i++)
-    {
-        lChange[0][i] = emptyPoint;
-        lChange[1][i] = emptyPoint;
+    bool retVal = false;
```

Comment:
Invent better name, pls for retVal
`flag ? labyrinth.addSnake(&snake) : false;`